client/src/components/BreakdownBox.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import Button from '@material-ui/core/Button';
+import StarRatings from 'react-star-ratings';
+
+function BreakdownBox ({avgRating}) {
+  return (
+      <div class="app-component breakdown-box">
+        <div class="breakdown-box-top-bar">
+          <h2 class="breakdown-box">REVIEWS</h2>
+          <Button className="write-review-button" children="Write a Review"/>
+        </div>
+        <div class="breakdown-box-bottom">
+          <div class="breakdown-box-section">
+            <h4>Rating Snapshot</h4>
+            <p>Select a row below to filter reviews.</p>
```

Comment:
should there be here the Rating snapshot component and some event handlers if a user for example clicks on on 4 stars reviews?

Reply:
great idea, I am going to make this a tickte to refactor this into it's own component so that functionality can be added